src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
+
+As my spirit approached the outskirts of the Dark Carnival, a hooded figure came out to greet me. He pulled his hood down. It was a skeleton of Shaggy 2 Dope and he had a message:
+
+“You dumbass bitch. This isn’t how the game ends! Get back in there and don’t go doin’ stupid shit again.”
+
+The Shaggy skeleton made the shocker with his bone hands.
+
+[[I felt time swirl and morph around me|The Gutter - Bar - Belinda]].
+
+::The Gutter - Bar - Darryl
+I sat down next to Darryl. He was a dipshit, but at least he wasn’t loud about it. He also didn’t care for Ron, which to me qualified as having perfect judgment. I started right in:
+
+>>[[“What’s the deal with Toe? He not talking today?”|The Gutter - Bar - Darryl game]]
+>>[[“What do you dream about, Darryl?"|The Gutter - Bar - Darryl dream]]
+>>[[“Is Toe his real name?”|The Gutter - Bar - Darryl toe]]
+>>[[“Why don’t you like Ron?”|The Gutter - Bar - Darryl ron]]
+>>[["See ya, Darryl."|The Gutter - Bar hub]]
+
+::The Gutter - Bar - Darryl game
+Darryl shook his head. “Eh, he never says much. About the only thing he ever does is pour and play games.”
+
+“What kind of games?” I asked.
+
+“Oh, anything. He had a collection, but some little shits recently stole most of it. It’s gotten to the point where he’s only got the stuff for pool and he won’t give it up to anybody.”
+
+[[I took note of Darryl’s observations|The Gutter - Bar - Darryl]].
+
+::The Gutter - Bar - Darryl dream
+Darryl’s eyes widened and he gulped. He let his vision linger on the edge of the bar for a moment before he spoke.
+
+“Mostly Grimace, the dark purple beast spoken of in the Book of Revelations that you would get with every McDonalds happy meal. When Grimace comes on his dark chariot driven by McNuggets, those of us who haven’t supped of the fry and the flesh weekly will be punished. It will not be swift. It will be the opposite of mercy.”
+
+Darryl cleared his throat, as if he’d just spoken several sentences by accident.
+
+“Oh, and last night I had a sexy dream about the mom from Family Guy. Don’t tell nobody.”
+
+[[I made a mental note to never ask Darryl a personal question again|The Gutter - Bar - Darryl]].
+
+::The Gutter - Bar - Darryl toe
+“Well...” Darryl paused for dramatic effect, or maybe he forgot the story.
+
+“They call him Toe because he has none. He cut them off and tried to eat them during a nasty trip, the story goes.
+
+“I’ve heard he used to love to have chicks suck his toes and now he pays to have them suck the air where they used to be.”
+
+Toe loomed over us both and we shrank. He nodded gravely and cleared Darryl’s empty glass.
+
+As he was turning away he let out a low rumble. “It’s all true.”
+
+[[I had solved another mystery. Good practice, I guess|The Gutter - Bar - Darryl]]?
+
+::The Gutter - Bar - Darryl ron
+“That’s between me, Ron, and the packet of hot sauce he tried to shove down my trousers as a prank.” Darryl said plainly.
+
+“Oh, dangit.” Darryl shook his head. “I goofed.”
```

Comment:
This was a little confusing to me. What was the goof? Was he not supposed to mention what happened?

Reply:
Yeah, it took me a re-read too, but the joke is that he's saying the reason he doesn't like Ron between him & etc., but then the etc. includes the reason.

Reply:
I think I overused goof here. I rewrote so he says "I told" and you laugh at his "mistake".